dll/win32/shell32/dialogs/recycler_prop.cpp

```diff
@@ -146,6 +147,11 @@ InitializeRecycleBinDlg(HWND hwndDlg, WCHAR DefaultDrive)
                                 defIndex = itemCount;
                                 pDefault = pItem;
                             }
+
+                            dwSize = sizeof(DWORD);
+                            RegGetValueW(HKEY_CURRENT_USER, L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Policies\\Explore", L"ConfirmFileDelete", RRF_RT_DWORD, NULL, &pItem->dwConfirmDelete, &dwSize);
```

Comment:
You should handle failure, in case the value doesn't exist, and store a fallback value in `dwConfirmDelete` if so.

Reply:
Okay, this could be resolved by RRF_ZEROONFAILURE, tomorrow i will submit it, that ensures the default value is zero

Reply:
Done